src/foam/core/Enum.js

```diff
@@ -328,6 +328,11 @@ foam.CLASS({
       class: 'StringArray',
       name: 'extraClasses',
       generateJava: false
+    },
+    {
+      class: 'GlyphProperty',
+      name: 'glyph',
+      value: null
```

Comment:
do you need to set default to null here?

Reply:
Yes, I think it defaults to an empty string and it wouldn't build with that.

Reply:
In types.js where you added GlyphProperty, you can add `value: null` there so that you won't have to specify it anytime you use the GlyphProperty class

Reply:
I only need it for enums because I think they need a default value